docs/config.md

```diff
@@ -146,6 +146,7 @@ due to lack of stable semantic convention.
 | `STACKEXCHANGEREDIS`  | [StackExchange.Redis](https://www.nuget.org/packages/StackExchange.Redis) **Not supported on .NET Framework**                                                                                   | ≥2.0.405 < 3.0.0   | source & bytecode    | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
 | `WCFCLIENT`           | WCF                                                                                                                                                                                             | *                  | source & bytecode    | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
 | `WCFSERVICE`          | WCF **Not supported on .NET**.                                                                                                                                                                  | *                  | source & bytecode    | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| `KAFKA`               | [Confluent.Kafka](https://www.nuget.org/packages/Confluent.Kafka)                                                                                                                               | ≥1.4.0 < 3.0.0     | bytecode             | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
```

Comment:
alphabetical order?

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3055/commits/eb633515a69b990c28e56b3f6d303ce4d640301d